src/schema/system/time.js

```diff
@@ -0,0 +1,50 @@
+import gravity from "lib/apis/gravity" // Uncached
+
+import { loaderOneOffFactory } from "lib/loaders/api/loader_one_off_factory"
+
+import {
+  GraphQLString,
+  GraphQLBoolean,
+  GraphQLInt,
+  GraphQLObjectType,
+} from "graphql"
+
+const SystemTimeType = new GraphQLObjectType({
+  name: "SystemTime",
+  fields: () => {
+    return {
+      day: { type: GraphQLInt },
+      wday: { type: GraphQLInt },
+      month: { type: GraphQLInt },
+      year: { type: GraphQLInt },
+      hour: { type: GraphQLInt },
+      min: { type: GraphQLInt },
+      sec: { type: GraphQLInt },
+      dst: { type: GraphQLBoolean },
+      unix: { type: GraphQLInt },
+      utc_offset: { type: GraphQLInt },
+      zone: { type: GraphQLString },
+      iso8601: { type: GraphQLString },
+    }
+  },
+})
+
+const SystemTime = {
+  type: SystemTimeType,
+  description:
+    "Gravity system time, necessary for synchronizing device clocks.",
+  resolve: async () => {
+    try {
+      const result = await loaderOneOffFactory(
+        gravity,
+        "gravity",
+        "system/time"
+      )
+      return result
+    } catch (e) {
```

Comment:
I believe this will override our default error handlers (which will notify Sentry, etc.). Is there a reason for handling this with a special `catch`?

I would basically just follow the pattern for the loader/factory stuff, and then the resolver would basically just be a `return systemTimeLoader()`, with no special error handling required here.

Reply:
I think using the factory will help.